pkg/workload/workload.go

```diff
@@ -244,6 +245,24 @@ func UpdateStatusIfChanged(ctx context.Context,
 	return UpdateStatus(ctx, c, wl, conditionType, conditionStatus, reason, message, managerPrefix)
 }
 
+func UnsetAdmissionWithCondition(
+	ctx context.Context,
+	c client.Client,
+	wl *kueue.Workload,
+	conditionStatus metav1.ConditionStatus,
```

Comment:
remove this parameter. It should always be false.

Reply:
done